src/core/Microsoft.PowerToys.Settings.UI/ViewModels/KeyboardManagerItemSelector.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.PowerToys.Settings.UI.ViewModels
+{
+    public class KeyboardManagerItemSelector : DataTemplateSelector

```

Comment:
This is intend for adding logic to selected which data template to use in list view since there can be multiple cases like (1:1 keys), (2:2 keys, 2:3, 3:3,so on) for shortcuts. I will remove this from this PR.